src/UKHO.S100PermitService.Common/Services/PermitService.cs

```diff
@@ -137,7 +137,7 @@ private void CreatePermitXml(string userPermit, string title, List<Products> pro
         }
 
         [ExcludeFromCodeCoverage]
-        private List<Products> GetProductsList(List<HoldingsServiceResponse> holdingsServiceResponse, IEnumerable<ProductKey> decryptedProductKeys, string hardwareId, string upnTitle)
+        private List<Products> GetProductsList(IEnumerable<HoldingsServiceResponse> holdingsServiceResponse, IEnumerable<ProductKey> decryptedProductKeys, string hardwareId, string upnTitle)
```

Comment:
The method should return an IEnumerable<Products> as well

Reply:
Resolved